spec/safe_migration_spec.rb

```diff
@@ -1278,5 +1297,52 @@ def up
         end
       end
     end
+
+    describe "disable_ddl_transaction" do
+      context "when instance variable is unset" do
+        let(:migration) do
+          Class.new(migration_klass).new
+        end
+
+        context "when default is true" do
```

Comment:
Small thing here, but I was thinking the word `config` or or `configuration` would be clearer than `default`. Since the term `default` is already used in the README and config to denote that it's set to `false` out of the box.

Reply:
I've gone for `"when configured globally to be (true|false)"`